structure_tests/tests/structure_test_tests.sh

```diff
@@ -34,8 +34,7 @@ popd
 
 # Run the debian tests, they should always pass on latest
 envsubst < cloudbuild.yaml.in > cloudbuild.yaml
-gcloud container builds submit . --config=cloudbuild.yaml
-if [ "$?" -gt "0" ]
+if [ "$(gcloud container builds submit . --config=cloudbuild.yaml)" -gt "0" ]
```

Comment:
This command is functionally different than what it was prior.  This is saying, take the output of "gcloud container builds submit . --config=cloudbuild.yaml" and check that it is >= "0".  The old check was analyzing the return codes.